docs/tutorials/simple-date-clock-application.adoc

```diff
@@ -0,0 +1,435 @@
+----
+
+To check if the creation of the cluster went right, run `kubectl cluster-info`.
+
+The setup of the cluster is not yet entirely finished.
+To conclude this, you need to enable an addon.
+Kubernete comes with various addon to deal with various aspects of your cluster.
+As the app we will develop/run are sometimes webapps, they need to be exposed "outside" of your cluster.
+The https://kubernetes.io/docs/concepts/services-networking/ingress/[`ingress` addon] helps in exposing HTTP/HTTPS routes from outside your cluster to services inside your cluster.
+Its activation is fairly simple using minikube.
+
+.Activating `ingress` addon
+[source,bash]
+----
+minikube addons enable ingress
+----
+
+Finally, it's important to activate some environement variables *in each shell you'll use to run commands*.
+Those variable will connect the different tools (especially skaffold) to use the docker environement inside your minikube cluster to build your apps images.
+
+.Activating the different environement variables
+[source,bash]
+----
+minikube docker-env | source
+# OR
+eval $(minikube docker-env)
+----
+
+
+== Run the projects and examples from the cloud-harness repository
+
+To be able to build, locally deploy and run the existing app (or any app you'll develop), you have to generate a specific `helm` configuration that will be use by skaffold to build/deploy/run your apps.
+The generation of those `helm` artifacts is done using `harness-deployment` with specific options.
+The next snippet show how to generate the `helm` configuration, disabling TLS configuration, enabling the local environement, selecting the `azathoth` namespace to deploy the app, and run it for the `azathoth.local` domain (you can use whatever domain name you want).
+The namespace can be changed depending on in which namespace you want to deploy your app in your cluster.
+This configuration is generated for *all* the app present in the `applications` folder in the `cloud-harness` repository.
+
+Depending on the namespace you use for the generation of your configuration, you need to create it yourself.
+Create the namespace first, then generate the deployment configuration.
+
+.Creating the namespace for your minikube cluster
+[source,bash]
+----
+kubectl create ns azathoth
+----
+
+[IMPORTANT]
+If you don't create the namespace, the deployment will fail!
+
+
+.Generating the deployment/configuration artifacts for all apps/services
+[source,bash]
+----
+# ran from the cloud-harness repository root
+harness-deployment  . -u -dtls -l -d azathoth.local -e local -n azathoth.local
+----
+
+In the state of the repository I have on my machine, the apps and services that will be deployed and that `harness-deployment` generated the configuration for are:
+
+* samples,
+* jupyterhub,
+* sentry,
+* accounts,
+* common,
+* volumemanager,
+* argo,
+* workflows,
+* notifications,
+* events.
+
+As you can see, some of those projects are services and not app per-say.
```

Comment:
per se?

Reply:
Damn, you're right! I always mispelled it...